components/ram.c

```diff
@@ -51,4 +51,78 @@ ram_used(void)
 	       bprintf("%f", (float)(total - free - buffers - cached) / 1024 / 1024) :
 	       NULL;
 }
+#elif defined(__OpenBSD__)
+#include <sys/types.h>
+#include <sys/sysctl.h>
+
+#include <stdbool.h>
+#include <stdio.h>
```

Comment:
Move `<stdio.h>` up as it is common with `__linux__` and remove it from the `__linux__` block. Saves a few LOC.